pygsti/algorithms/germselection.py

```diff
@@ -169,19 +244,31 @@ def find_germs(target_model, randomize=True, randomization_strength=1e-2,
             'comm': comm,
             'mem_limit': mem_limit,
             'profiler': profiler,
-            'num_nongauge_params': num_nongauge_params
+            'num_nongauge_params': num_nongauge_params,
+            'num_gauge_params': num_gauge_params,
```

Comment:
Does this function need both num_nongauge_params and num_gauge_params as parameters?  It seems like the latter may not be used, in which case it'd probably be good to remove it?  I may be missing something here though.

Reply:
Probably not. I think the value of num_gauge_params is used exactly once and just for convience (i.e. it could just as easily be replaced by num_params-num_nongauge_params). I must've had a reason for wanting this at some point, but that looks to be lost to time. I'll add refactoring this to remove that argument to my to-do list.

Reply:
Just pushed a refactor for this.